examples/single_modality/clustering/graphsc.py

```diff
@@ -6,42 +6,36 @@
 import numpy as np
 import torch
 
+from dance.data import Data
 from dance.datasets.singlemodality import ClusteringDataset
 from dance.modules.single_modality.clustering.graphsc import *
-from dance.transforms.graph_construct import make_graph
+from dance.transforms.graph_construct import cell_gene_graph
 from dance.transforms.preprocess import filter_data
+from dance.utils import set_seed
 
 
 def pipeline(**args):
-    data = ClusteringDataset(args['data_dir'], args['dataset']).load_data()
-    X = data.X
-    Y = data.Y
+    adata, labels = ClusteringDataset(args['data_dir'], args['dataset']).load_data()
+    adata.obsm["labels"] = labels
+    adata = adata[:100, :]
+    data = Data(adata, train_size=adata.n_obs)
+    data.set_config(label_channel="labels")
```

Comment:
Could you combine this with the set_config below? "labels" channel might not be needed in the graph here